parm/config/config.resources

```diff
@@ -680,47 +700,52 @@ elif [[ ${step} = "arch" || ${step} = "earc" || ${step} = "getic" ]]; then
     eval "export npe_${step}=1"
     eval "export npe_node_${step}=1"
     eval "export nth_${step}=1"
-    eval "export memory_${step}=2048M"
+    eval "export memory_${step}=4096M"
```

Comment:
It may just be a one-off case.  I haven't seen this be a problem on any other system.  We had a discussion on this [here](https://github.com/NOAA-EMC/global-workflow/pull/1301#issuecomment-1468393293) as well.

Reply:
Ah yes, thanks for the reminder of the prior convo. Going to 4GB is ok. :)